<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddManager</class>
 <widget class="QDialog" name="AddManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Screenshot 2024-07-05 130354.png);
border-radius:15%;
color: rgb(0, 100, 250);
font: 10pt &quot;MV Boli&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Addmanagername">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 75 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Add Manger MENU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddmanagerID">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Departeman-ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddmanagerDid">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddmanagerPhone">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>National Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddmanageNcode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hours of Work:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddmanagerHW">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 50, 250);
font: 14pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>480</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
